runtime/kusama/src/lib.rs

```diff
@@ -1213,6 +1213,13 @@ impl parachains_initializer::Config for Runtime {
 	type WeightInfo = weights::runtime_parachains_initializer::WeightInfo<Runtime>;
 }
 
+impl parachains_disputes::Config for Runtime {
+	type Event = Event;
+	type RewardValidators = ();
+	type PunishValidators = ();
```

Comment:
is there a TODO to populate this?

Reply:
Yes

Reply:
@ordian is working on it

Reply:
These changes are probably going to land in 0.9.14. Not blocking this PR.